README.md

```diff
@@ -11,13 +11,13 @@ Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code) (also included below with a comment).
 
-This workflow requires two platforms.  One is USS which is the target platform.  The second is a generic linux
+This workflow requires two platforms.  One is z/OS® UNIX System Services (z/OS UNIX) which is the target platform.  The second is a generic linux
 AMD64 platform to get files normally acquired via `cipd`
 
 
-#### Linux AMD64 plaform setup
+#### Linux plaform (non z/OS® UNIX System Services (z/OS UNIX)) setup
```

Comment:
Suggest replacing:
Linux plaform (non z/OS® UNIX System Services (z/OS UNIX)) setup
by:
Steps on a non-z/OS UNIX platform

Reply:
@netskink it seems you've missed this suggested update as of the latest 66ba424.

Similarly please replace this line:
z/OS® UNIX System Services (z/OS UNIX) platform setup
by:
Steps on the target z/OS UNIX platform

(since these steps are not setting up the z/OS or non-z/OS UNIX platforms).

Reply:
Also please remove those 2 lines currently under "This workflow is done once":
```
$ git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
$ tar -cvf x.tar test tools third_party
```
and add them in a separate paragraph that follows it, with this text:

Do this afterwards and everytime you need to sync
```
$ cd $HOME/zopen/dev/v8base
$ git pull
$ git reset fce38915e4c7c73c0e7f77bb13e2a78514968d35 --hard
$ gclient sync
$ tar -cvf x.tar test tools third_party
```

Reply:
Also please remove the section showing the output from `tar -xvf ...`.